### ISSUE TITLE
Parse Server Crashes when Push Notification is Sent from Parse Dashboard

### ISSUE THREAD

**Issue by cobyanderson** [ParsePlatform/parse-server#1716]
#### Environment Setup

- Server: parse-server 2.2.7, running remote on Heroku
- Database: mongoDB using mLab

#### Steps to reproduce

- Each time I try to send a push notification from my parse server (on heroku) by using the parse dashboard (hosted locally) the parse server crashes. 
- I only have one installation, configured exactly according to the parse server push notification setup docs. 

Here I set up my api (the all caps keys are replaced with my own)
![image](https://cloud.githubusercontent.com/assets/13067470/15090114/92961a52-13ce-11e6-9146-3b627d62abe5.png)

Any reason why this would be failing?

**Comment by drew-gross** [comment]
To help us debug this, can you include the exact request and response in Parse Server? (you can see this in the logs using `VERBOSE=1` environment variable)

**Comment by cobyanderson** [comment]
![image](https://cloud.githubusercontent.com/assets/13067470/15100127/3d3c0d68-151e-11e6-8f28-87432f0f71e5.png)
I set the verbose variable in Heroku's config vars and the error log claimed first that my certificate's file could not be found. To fix this, I moved the certificate to the my parse server's files to be uploaded to Heroku, and used `__dirname + 'NAMEOFCERTIFICATE.p12` to make sure the path was correct. I ran again, and this time the logs seemed to claim my certificate was not vaild. I followed this solution (https://github.com/ParsePlatform/parse-server/issues/1282) and created a new development certificate, and, finally, I was able to send push notifications.
